apps/seven/app/root.tsx

```diff
@@ -52,6 +52,18 @@ export async function loader({ params, request }: Route.LoaderArgs) {
       cli.getSite(),
     ]);
 
+    const listingBlocks: { id: string; block: ListingBlockFormData }[] =
```

Comment:
yes, this was the idea, however in the future we should have a generic "visitor" that executes specific extra calls if needed. Using an utilitity for that then call them all and execute them like the `rootLoaderDataUtilities`.

Reply:
@sneridagh ok, done :)

I was not very comfortable using a very loose typing for the utility method (`(...args: any[]) => any`), so I have created a more precise typing, and default to the original as `DummyUtilityMethod`.

That way, you can do things like:
```
config.getUtilities<ContentSubRequestUtilityMethod>({
  type: 'rootContentSubRequest',
});
```
and you know for sure what you get.
That's safer.

Ideally, we should bind the typing with the utility type, like when calling `getUtilities` with `type: 'rootContentSubRequest'` then we should mandatorily obtain some `ContentSubRequestUtilityMethod`, but that would change the `setUtility` and `getUtilities` signatures, so maybe that should be done in another PR.